apps/resourses.txt

```diff
@@ -13,4 +13,9 @@ https://loremipsum.io/
 format = '%Y-%m-%d %H:%M:%S'
```

Comment:
what's the use of this file?

Reply:
Hmm, I don't think this file should be included in master branch. I'm removing it.